macros/math/PGnumericalmacros.pl

```diff
@@ -384,12 +390,12 @@ sub javaScript_cubic_spline {
 </NOSCRIPT>
 END_OF_JAVA_TEXT
 
-	$output_str;
+	return $output_str;
 }
 
 =head2 Numerical Integration methods
 
-=head3 Integration by Left Hand Sum
+=head3 lefthandsum (Left Hand Riemann Sum)
```

Comment:
Please move the parenthetical below.  So make this
```suggestion
=head3 lefthandsum

 Left Hand Riemann Sum
```
Either that or just drop it.  The method name is self explanatory, and the explanation below tells what it is anyway.

Do the same for `righthandsum` and `inv_romberg`.  The parentheticals don't look good in the sidebar.